WcaOnRails/config/locales/en.yml

```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
+      'regulations1':
```

Comment:
I was actually going to drop a message about that when I saw your review!
The key names don't matter, only the order of the keys **in English** does (I mentioned it [here](https://github.com/thewca/worldcubeassociation.org/pull/1097#issue-201022738)).

The `I18n.t` fetches English keys and preserves the order, and the `deep_merge` just override the ones already existing (meaning that even if foreign locales are in the wrong order, they are still displayed in the English order).